modules/rational-modeline.el

```diff
@@ -0,0 +1,158 @@
+;;;; rational-modeline.el --- rational module for setting up the modeline  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022
+;; SPDX-License-Identifier: MIT
+
+;; Authors: System Crafters Community, David "System Crafters" Wilson, Erik Lundstedt
```

Comment:
I don't think I have heard of @daviwil call himself _David "System Crafters" Wilson_ 🤔

Reply:
Yes, should probably update the comment.